src/components/theme.ts

```diff
@@ -57,7 +57,7 @@ const space = {
 
 const fonts = {
   primary: "Helvetica Neue, sans-serif",
-  secondary: "SangBleu, sans-serif",
+  secondary: "SangBleuOGSerif-Light, sans-serif",
```

Comment:
why did this get changed back? keeping it as SangBleu works for me. Can you check your end on the home page "/"

SangBleu:
![image](https://user-images.githubusercontent.com/11725595/97861491-299a8800-1d04-11eb-9181-d5e607e295a5.png)

SangBleuOGSerif-Light:

![image](https://user-images.githubusercontent.com/11725595/97861524-34551d00-1d04-11eb-8562-d692964138f0.png)

Reply:
This was the only fix for me to get the font to work on all pages.

SangBleuOGSerif-Light:
<img width="1440" alt="Screenshot 2020-11-02 at 12 17 58" src="https://user-images.githubusercontent.com/53219789/97862421-9a8e6f80-1d05-11eb-8b66-79c9d32581bc.png">